solidity/dashboard/src/css/typography.less

```diff
@@ -101,24 +101,13 @@ a {
   color: @secondary-color;
   position: relative;
 
-  &.arrow-link {
+  &.external {
     &::after {
-      content: '';
-      position: absolute;
-      width: 1.25em;
-      height: 1.25em;
-      background-size: 1.25em 1.25em;
-      top: 50%;
-      margin-top: -0.625em;
-      right: calc(-1.25em - 2px);
-      background-image: url("../static/svg/link-arrow.svg");
-
-    }
-    
-    &.grey {
-      &::after {
-        background-image: url("../static/svg/link-arrow-grey.svg");
-      }
+      // north east arrow code
+      content: '\2197';
+      margin-left: 0.25rem;
```

Comment:
seems that spacing doesn't work. We also need `white-space: pre` pls [see].

Reply:
🤔 but this isn't at the end of a line, right? I would expect a space by itself not to work, but a space followed by the unicode character to render as expected.

Reply:
Ah I see, this doesn't work because of the `inline-block` display, which makes the space be at the beginning of a block-level item, and thus unrendered. I think the right move here is to buck the design and underline the arrow. We can discuss with Liz + Michael on the back end, but right now that's the most consistent approach with how we've handled these elsewhere, so let's keep it consistent and then decide if we want to consistently switch to not underlining.